static/package.json

```diff
@@ -44,6 +44,7 @@
     "vue-tsc": "^0.31.4"
   },
   "prettier": {
-    "htmlWhitespaceSensitivity": "ignore"
+    "htmlWhitespaceSensitivity": "ignore",
+    "endOfLine": "auto"
```

Comment:
What does it do? Should we add it to the vscode config as well?

Reply:
Default is "lf" so with "auto" it will not format line endings to lf and the linter will thus also not complain about crlf. Initially I wanted to have this setting locally so the linter output is ok, but this will also apply for the ci now. For safety I will remove it again, although all files will be converted to lf when pushing by git.